python-crud-google-spreadsheet/README.md

```diff
@@ -1,8 +1,5 @@
 # Python3 CRUD + Google Spreadsheet Integration With Functions
-Create, Read, Update, and Delete entries using a google spreadsheet for storage
-
-# What does it do?
-Creates a simple crud service using only binaris functions written in python3.
+Create, Read, Update, and Delete entries using a Google Sheet for storage
 
 # Deploy me!
 
```

Comment:
**G**oogle

Reply:
Instead of:
Get a Binaris API Key and a Google project + credentials and a usable spreadsheet from Google Docs.

Consider:
Get a Binaris API Key, a Google project with credentials, and a new spreadsheet in Google Drive.

Reply:
I don't think this really changes anything.